bindings/go/README.md

```diff
@@ -17,12 +17,38 @@ provided there is enough buffer size to accomodate several of such blocks in one
 Read()/Write() request. (For example, see the source code of `mcp` utility and its `-bsz`
 option.)
 
-To use `go/mio` in your app 1) install motr-devel pkg and 2):
+## Installation
+
+Use go get, make sure you using Go version with [modules](https://github.com/golang/go/wiki/Modules) support.
+
+```Shell
+go get github.com/Seagate/cortx-motr/bindings/go
+```
+
+Next install `motr-devel` pkg and then import into sources.
```

Comment:
motr and motr-devel - both of them should be installed.

Reply:
ok, I will add this in.